### PR TITLE
Update QAT INT8 2.0 doc 

### PR DESCRIPTION
**Pull request by lidanqing-intel**
Update document for 2.0. 
Benchmark data in the table will be updated soon.

### CODE REVIEW

File: paddle/fluid/inference/tests/api/int8_mkldnn_quantization.md [hunk 1]
```diff
@@ -64,14 +64,32 @@ We provide the results of accuracy and performance measured on Intel(R) Xeon(R)
...
+cd /PATH/TO/PADDLE/
+python paddle/fluid/inference/tests/api/full_ILSVRC2012_val_preprocess.py --local --data_dir=/PATH/TO/USER/DATASET --output_file=/PATH/TO/OUTPUT/BINARY
 ```
 
-Then the ILSVRC2012 Validation dataset will be preprocessed and saved by default in `$HOME/.cache/paddle/dataset/int8/download/int8_full_val.bin`
+Available options in the above command and their descriptions are as follows:
+- **No parameters set:** The script will download the ILSVRC2012_img_val data from server and convert it into a binary file.
+- **local:** Once set, the script will process user local data.
+- **data_dir:** Path to user local dataset. Default value: None.
+- **label_list:** Path to path_to_one_image-path_to_its_label list file. Default value: `val_list.txt`.
```
Comment (wojtuss) [inline paddle/fluid/inference/tests/api/int8_mkldnn_quantization.md]: The description here is unclear.

File: paddle/fluid/inference/tests/api/int8_mkldnn_quantization.md [hunk 2]
```diff
@@ -108,30 +126,30 @@ MODEL_NAME=googlenet, mobilenetv1, mobilenetv2, resnet101, resnet50, vgg16, vgg1
...
-                                         --ap_version=11point \\
+cd /PATH/TO/PADDLE
+python paddle/fluid/inference/tests/api/full_pascalvoc_test_preprocess.py --local --data_dir=/PATH/TO/USER/DATASET --img_annotation_list=/PATH/TO/ANNOTATION/LIST --label_file=/PATH/TO/LABEL/FILE --output_file=/PATH/TO/OUTPUT/FILE
 ```
-Then the user dataset will be preprocessed and saved by default in `/PATH/TO/PADDLE/build/third_party/inference_demo/int8v2/pascalvoc_small/pascalvoc_small.bin`
+Available options in the above command and their descriptions are as follows:
+- **No parameters set:** The script will download the full pascalvoc test dataset and preprocess and convert it into a binary file.
+- **local:** Once set, the script will process user local data.
+- **data_dir:** Path to user local dataset. Default value: None.
+- **img_annotation_list:** Path to image_path-image categ list. Default value: `test_100.txt`.
```
Comment (wojtuss) [inline paddle/fluid/inference/tests/api/int8_mkldnn_quantization.md]: Same here.

**Comment by lidanqing-vv** [comment]
@juncaipeng  Juncai, after QA Team verified this document, please let me know. I need to cherry-pick it to 2.0. Thanks.

File: paddle/fluid/inference/tests/api/int8_mkldnn_quantization.md (hunk 1 above)
Comment (lidanqing-vv) [reply paddle/fluid/inference/tests/api/int8_mkldnn_quantization.md]: changed

File: paddle/fluid/inference/tests/api/int8_mkldnn_quantization.md (hunk 2 above)
Comment (lidanqing-vv) [reply paddle/fluid/inference/tests/api/int8_mkldnn_quantization.md]: changed

**Pull request merged**